src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -149,14 +149,22 @@ public function text($message)
     {
         $this->autoPrependText();
 
-        if (!is_array($message)) {
-            $this->writeln(sprintf(' // %s', $message));
-
-            return;
+        $messages = is_array($message) ? array_values($message) : array($message);
+        foreach ($messages as $message) {
+             $this->writeln(sprintf(' %s', $message));
         }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function comment($message)
```

Comment:
This method is not defined in the parent class. So inheritdoc makes no sense. Also I guess this method should be part of StyleInterface. But I don't really see the point of this interface either.